loopback-example/.vscode/settings.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
do we need to commit .vscode ?

Reply:
it was auto-generated by `lb4` and added as example